lib/graticule/geocoder/mapbox.rb

```diff
@@ -1,10 +1,11 @@
-module Graticule
-  module Geocoder
+require 'json'
+module Graticule #:nodoc:
+  module Geocoder #:nodoc:
     class Mapbox < Base
-      BASE_URL = "http://api.tiles.mapbox.com"
 
       def initialize(api_key)
         @api_key = api_key
+        @url = "http://api.mapbox.com"
```

Comment:
It doesn't look like setting `@url` is necessary. The only time it's referenced is in `make_url` where it is not used, but reassigned. The `@url` in `make_url` could instead be a local variable, but I personally like the approach of pulling the service's base url to to the top of the class so it's not buried in the details.